### PR TITLE
pveperf: add page

### PR DESCRIPTION
**Pull request by einverne**
**Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/pveperf.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# pveperf
+
+> Proxmox benchmark.
```
Comment (marchersimon) [inline pages/common/pveperf.md:3]: Could you please elaborate on this a bit more? As someone who doesn't know Proxmox I have no idea what this command does.

Comment (einverne) [reply pages/common/pveperf.md:3]:
```suggestion
> A benchmarking tool in proxmox server. Gather CPU/hard disk performance data on the hard disk mounted at PATH (/ is used as default).
```

File: pages/common/pveperf.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# pveperf
+
+> A benchmarking tool in proxmox server. Gather CPU/hard disk performance data on the hard disk mounted at PATH (/ is used as default).
```
Comment (navarroaxel) [inline pages/common/pveperf.md:3]:
```suggestion
> A benchmarking tool in Proxmox Server. Gather CPU and hard disk performance data on the hard disk mounted at PATH (`/` is used as default).
```

File: pages/common/pveperf.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+> More information: <https://pve.proxmox.com/pve-docs/pveperf.1.html>.
+
+- Show CPU/hard disk performance data on the hard disk mounted at `/`:
```
Comment (navarroaxel) [inline pages/common/pveperf.md:6]: Should this be a `for`? :thinking: 
```suggestion
- Show CPU/hard disk performance data for the hard disk mounted at `/`:
```

File: pages/common/pveperf.md (hunk 2 above)
Comment (marchersimon) [reply pages/common/pveperf.md:3]: I don't think we need that much detail in the general description.
```suggestion
> A benchmarking tool in Proxmox Server. Gather CPU and hard disk performance data for the hard disk.
```

File: pages/common/pveperf.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
+# pveperf
+
+> A benchmarking tool in Proxmox Server. Gather CPU and hard disk performance data for the hard disk.
+> More information: <https://pve.proxmox.com/pve-docs/pveperf.1.html>.
+
+- Show CPU/hard disk performance data for the hard disk mounted at `/`:
```
Comment (marchersimon) [inline pages/common/pveperf.md:6]:
```suggestion
- Show CPU and hard disk performance data for the hard disk mounted at `/`:
```

**Review by marchersimon — APPROVED** [review]
Thanks @einverne 
I just left a small suggestion below

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**